dataikuapi/dss/ml.py

```diff
@@ -230,28 +233,7 @@ def use_feature(self, feature_name):
         self.get_feature_preprocessing(feature_name)["role"] = "INPUT"
 
     def get_algorithm_settings(self, algorithm_name):
-        """
-        Gets the training settings for a particular algorithm. This returns a reference to the
-        algorithm's settings, not a copy, so changes made to the returned object will be reflected when saving.
-
-        This method returns a dictionary of the settings for this algorithm.
-        All algorithm dicts have at least an "enabled" key in the dictionary.
-        The 'enabled' key indicates whether this algorithm will be trained
-
-        Other settings are algorithm-dependent and are the various hyperparameters of the 
-        algorithm. The precise keys for each algorithm are not all documented. You can print
-        the returned dictionary to learn more about the settings of each particular algorithm
-
-        Please refer to the documentation for details on available algorithms.
-
-        :param str algorithm_name: Name (in capitals) of the algorithm.
-        :return: A dict of the settings for an algorithm
-        :rtype: dict 
-        """
-        if algorithm_name in self.__class__.algorithm_remap:
-            algorithm_name = self.__class__.algorithm_remap[algorithm_name]
-
-        return self.mltask_settings["modeling"][algorithm_name.lower()]
+        raise NotImplementedError()
```

Comment:
Shouldn't we deprecate this method, rather than breaking it ? Note that it currently also breaks `set_algorithm_enabled`